### ISSUE TITLE
Ability to fire POST when a record has been added/updated

### ISSUE THREAD

**Issue by ahankinson** [DDMAL/misirlou#98]
If a manifest has been added or updated, we should have the ability to fire a POST request to a configurable listener with a URL to the record in the body of the request.

This will be needed when, for example, we add a new source to musiclibs and then want to trigger an OMR workflow in Rodan.

**Comment by agpar** [comment]
A listener that can send a post request to an outside service when a manifest is added/updated?

**Comment by ahankinson** [comment]
Yeah -- we'll want to be able to fire off a signal (or several signals?) to services that can do stuff with our manifests. The most obvious one is Rodan (see the linked issue) so that we can automatically OMR the stuff coming in.

**Comment by ahankinson** [comment]
We might also want listeners too so that when the Rodan workflow is done we can send back the results of the workflow and incorporate it into the Musiclibs index.

**Comment by agpar** [comment]
I've added the scaffolding to do whatever sort of work we need to do when a manifest is updated/added in e83ec1dba796f8fa65748f4a4c93483d110696cd.

**Comment by ahankinson** [comment]
Could you please make sure it's documented well? This is one bit of functionality external to musiclibs that would benefit from having some human-readable documentation.

**Comment by agpar** [comment]
Right now it has no external functionality. I haven't worked out with Ryan exactly what sort of external call to make - it's just the scaffolding (a signal that is handled by a function that does nothing).

**Comment by agpar** [comment]
That is - the commit I linked:
1. Defines a signal.
2. Dispatches that signal in the correct situation.
3. Defines a signal handler, which currently does nothing.
